src/App.tsx

```diff
@@ -1,33 +1,128 @@
+import React, { useState } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodosAndUsers } from './types/todosAndUsers';
+import { User } from './types/user';
+
+function findUserById(userId: number): User | null {
+  return usersFromServer.find(user => userId === user.id) || null;
+}
+
+function addNewTodoId(todos: TodosAndUsers[]): number {
+  return Math.max(...todos.map(todo => todo.id)) + 1;
```

Comment:
no need to copy an array, is map a mutating method or not?
```suggestion
  return Math.max(todos.map(todo => todo.id)) + 1;
```